config/default.json

```diff
@@ -46,6 +46,7 @@
   },
   "jpegTileEstimatedSizeInBytes": 12500,
   "storageFactorBuffer": 1.25,
+  "validateStorage": true,
```

Comment:
create section of 
storageEstimation {
   "jpegTileEstimatedSizeInBytes": 12500,
  "storageFactorBuffer": 1.25,
  "validateStorage": true
}
rename validateStorage to validateStorageSize

Reply:
done